docs/source/metaclasses.rst

```diff
@@ -32,7 +32,7 @@ In Python 2, the syntax for defining a metaclass is different:
     class A(object):
         __metaclass__ = M
 
-Mypy also supports using :py:func:`six.with_metaclass`
+Mypy also supports using :py:func:`six.with_metaclass` and :py:func:`@six.add_metaclass <six.add_metaclass>`
```

Comment:
I have decided to extend the text as the `@six.add_metaclass` decorator is used in the subsequent code example as well.